modules/subscriptions/views.php

```diff
@@ -322,15 +338,31 @@ static function render_widget_subscription_form( $args, $instance, $subscribe_em
 						echo wpautop( $subscribe_text );
 					}
 					if ( $show_subscribers_total && $subscribers_total ) {
-						/* translators: %s: number of folks following the blog */
-						echo wpautop( sprintf( _n( 'Join %s other follower', 'Join %s other followers', $subscribers_total ), number_format_i18n( $subscribers_total ) ) );
+						?>
+						<div class="subscribe-count">
+							<p>
+							<?php
+							/* translators: %s: number of folks following the blog */
+							echo esc_html( sprintf( _n( 'Join %s other follower', 'Join %s other followers', $subscribers_total, 'jetpack' ), number_format_i18n( $subscribers_total ) ) );
+							?>
+							</p>
+						</div>
+						<?php
 					}
 					?>
-                    <p><input type="text" name="email" style="width: 95%; padding: 1px 2px"
-                              placeholder="<?php esc_attr_e( 'Enter your email address' ); ?>" value=""
-                              id="subscribe-field<?php if ( Jetpack_Subscriptions_Widget::$instance_count > 1 ) {
-						          echo '-' . Jetpack_Subscriptions_Widget::$instance_count;
-					          } ?>"/></p>
+                    <p>
+                    	<input type="text" name="email"
+                    		<?php if ( ! empty( $email_field_classes ) ) { ?>
+	                        	class="<?php echo esc_attr( $email_field_classes ); ?>"
```

Comment:
Should probably use `esc_attr_e()` consistently instead of `echo`.

Reply:
There's a lot of these, I'll update them all.

Reply:
The linter does not allow anything other string literals to be passed to these `_e` equivalents, so this would be a fair refactor. I think we should just leave this as is in this PR.